src/test/java/com/fasterxml/uuid/jug/UsageInfoTest.java

```diff
@@ -0,0 +1,73 @@
+package com.fasterxml.uuid.jug;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.Assert.assertEquals;
+
+public class UsageInfoTest {
```

Comment:
Seriously? Test for UsageInfo?! What possible point is there testing specific output.